### PR TITLE
feat: rebranding meaning BREAKING CHANGE

### PR DESCRIPTION
**Pull request by gautierlabarre**

### CODE REVIEW

**Review by syjust-alma — APPROVED** [review]

File: src/Widgets/EligibilityModal/EligibilityModal.module.css [hunk 1]
```diff
@@ -12,3 +12,26 @@
...
+}
+
+.verticalLine {
+  position: absolute;
+  top: 40px;
+  left: 5px;
+  width: 2px;
+  /* 140px is a magic number, to avoid seeing the line under the TotalBlock */
+  height: calc(100% - 140px);
+  background-color: var(--soft-blue);
```
Comment (ncordin) [inline src/Widgets/EligibilityModal/EligibilityModal.module.css:27]: i don't get what this soft-blue is for. i see the vertical line orange on figma

File: src/Widgets/EligibilityModal/MobileModal/index.tsx [hunk 2]
```diff
@@ -21,7 +21,7 @@ const MobileModal: FC<{ isSomePlanDeferred: boolean; cards?: Card[] }> = ({
     {children}
     <Info />
     {cards && <Cards cards={cards} />}
-    <Logo />
+    <AlmaLogo className={s.logo} width="75" />
```
Comment (ncordin) [inline src/Widgets/EligibilityModal/MobileModal/index.tsx:24]: i think we must precise `px` in 75px

File: src/Widgets/EligibilityModal/components/EligibilityPlansButtons/EligibilityPlansButtons.module.css [hunk 3]
```diff
@@ -4,42 +4,26 @@
...
-  border: 1px solid var(--gray-300);
-  border-radius: 8px;
-  font-family: 'Eina04', sans-serif;
-  font-size: 18px;
-  line-height: 150%;
+  min-width: 50px;
+  border: 1px solid var(--dark-gray);
+  text-transform: lowercase;
+  border-radius: 16px;
+  font-family: 'Argent CF', sans-serif;
```
Comment (ncordin) [inline src/Widgets/EligibilityModal/components/EligibilityPlansButtons/EligibilityPlansButtons.module.css:18]:
```suggestion
  font-family: 'Argent CF', serif;
```

File: src/Widgets/EligibilityModal/components/Info/Info.module.css [hunk 4]
```diff
@@ -3,26 +3,30 @@
...
-  height: 30px;
-  line-height: 30px;
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  border-radius: 50%;
-  background-color: var(--blue-300);
-  color: var(--white);
-  margin-right: 16px;
+  font-family: 'Argent CF', sans-serif;
```
Comment (ncordin) [inline src/Widgets/EligibilityModal/components/Info/Info.module.css:17]:
```suggestion
  font-family: 'Argent CF', serif;
```

File: src/Widgets/EligibilityModal/components/Title/Title.module.css [hunk 5]
```diff
@@ -1,23 +1,14 @@
 .title {
-  font-family: 'Eina04', sans-serif;
+  font-family: 'Argent CF', sans-serif;
```
Comment (ncordin) [inline src/Widgets/EligibilityModal/components/Title/Title.module.css:2]:
```suggestion
  font-family: 'Argent CF', serif;
```

File: src/Widgets/EligibilityModal/components/Schedule/index.tsx [hunk 6]
```diff
@@ -23,85 +17,15 @@ const Schedule: FC<{ currentPlan: EligibilityPlan }> = ({ currentPlan }) => {
...
-        </div>
         {isCredit && (
-          <p className={s.creditMessage}>
+          <div className={s.creditInfo}>
             <FormattedMessage
-              id="eligibility-modal.credit-commitment"
-              defaultMessage="Un crédit vous engage et doit être remboursé. Vérifiez vos capacités de remboursement
-              avant de vous engager."
+              id="credit-features.information"
+              defaultMessage="Un crédit vous engage et doit être remboursé. Vérifiez vos capacités de remboursement avant de vous engager."
```
Comment (Taskim) [inline src/Widgets/EligibilityModal/components/Schedule/index.tsx:26]: the margin left is too important

File: src/Widgets/PaymentPlans/index.tsx [hunk 7]
```diff
@@ -130,7 +130,7 @@ const PaymentPlanWidget: VoidFunctionComponent<Props> = ({
         data-testid="widget-button"
       >
         <div className={cx(s.primaryContainer, STATIC_CUSTOMISATION_CLASSES.eligibilityLine)}>
-          <LogoIcon className={s.logo} monochrome={monochrome} />
+          <AlmaLogo className={s.logo} color={monochrome ? '#1A1A1A' : undefined} />
```
Comment (Taskim) [inline src/Widgets/PaymentPlans/index.tsx]:
```suggestion
          <AlmaLogo className={s.logo} color={monochrome ? 'var(--off-black)' : undefined} />
```

**Review by Taskim — APPROVED** [review]

**Review by ncordin — APPROVED** [review]

File: src/Widgets/EligibilityModal/MobileModal/index.tsx (hunk 2 above)
Comment (gautierlabarre) [reply src/Widgets/EligibilityModal/MobileModal/index.tsx:24]: I don't think, no https://www.w3schools.com/tags/att_width.asp

File: src/Widgets/EligibilityModal/components/EligibilityPlansButtons/EligibilityPlansButtons.module.css (hunk 3 above)
Comment (gautierlabarre) [reply src/Widgets/EligibilityModal/components/EligibilityPlansButtons/EligibilityPlansButtons.module.css:18]: I'll fix this on a future release, if that's ok with you.

**Pull request merged**